presto-base-arrow-flight/src/test/java/com/facebook/plugin/arrow/TestingH2DatabaseSetup.java

```diff
@@ -0,0 +1,273 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.plugin.arrow;
+
+import com.facebook.airlift.log.Logger;
+import com.facebook.presto.common.type.Type;
+import com.facebook.presto.common.type.VarcharType;
+import com.facebook.presto.spi.ColumnMetadata;
+import com.facebook.presto.spi.ConnectorTableMetadata;
+import com.facebook.presto.spi.RecordCursor;
+import com.facebook.presto.spi.RecordSet;
+import com.facebook.presto.spi.SchemaTableName;
+import com.facebook.presto.tpch.TpchMetadata;
+import com.facebook.presto.tpch.TpchTableHandle;
+import com.google.common.base.Joiner;
+import io.airlift.tpch.TpchTable;
+import org.jdbi.v3.core.Handle;
+import org.jdbi.v3.core.Jdbi;
+import org.jdbi.v3.core.statement.PreparedBatch;
+import org.joda.time.DateTimeZone;
+
+import java.sql.Connection;
+import java.sql.Date;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Collectors;
+
+import static com.facebook.presto.common.type.BigintType.BIGINT;
+import static com.facebook.presto.common.type.BooleanType.BOOLEAN;
+import static com.facebook.presto.common.type.DateType.DATE;
+import static com.facebook.presto.common.type.DoubleType.DOUBLE;
+import static com.facebook.presto.common.type.IntegerType.INTEGER;
+import static com.facebook.presto.tpch.TpchMetadata.TINY_SCHEMA_NAME;
+import static com.facebook.presto.tpch.TpchRecordSet.createTpchRecordSet;
+import static com.google.common.collect.ImmutableList.toImmutableList;
+import static io.airlift.tpch.TpchTable.CUSTOMER;
+import static io.airlift.tpch.TpchTable.LINE_ITEM;
+import static io.airlift.tpch.TpchTable.NATION;
+import static io.airlift.tpch.TpchTable.ORDERS;
+import static io.airlift.tpch.TpchTable.PART;
+import static io.airlift.tpch.TpchTable.PART_SUPPLIER;
+import static io.airlift.tpch.TpchTable.REGION;
+import static io.airlift.tpch.TpchTable.SUPPLIER;
+import static java.lang.String.format;
+import static java.util.Collections.nCopies;
+
+public class TestingH2DatabaseSetup
+{
+    private static final Logger logger = Logger.get(TestingH2DatabaseSetup.class);
+    private TestingH2DatabaseSetup()
+    {
+        throw new UnsupportedOperationException("This is a utility class and cannot be instantiated");
+    }
+
+    public static void setup(String h2JdbcUrl) throws Exception
+    {
+        Class.forName("org.h2.Driver");
+
+        Connection conn = DriverManager.getConnection(h2JdbcUrl, "sa", "");
+
+        Jdbi jdbi = Jdbi.create(h2JdbcUrl, "sa", "");
+        Handle handle = jdbi.open(); // Get a handle for the database connection
+
+        TpchMetadata tpchMetadata = new TpchMetadata("");
+
+        Statement stmt = conn.createStatement();
+
+        // Create schema
+        stmt.execute("CREATE SCHEMA IF NOT EXISTS testdb");
```

Comment:
Let's call this tpch instead.
```suggestion
        stmt.execute("CREATE SCHEMA IF NOT EXISTS tpch");
```

Reply:
Updated to use tpch